openstack/objectstorage/v1/objects/testing/fixtures_test.go

```diff
@@ -94,7 +94,7 @@ var ExpectedListSubdir = []objects.Object{
 
 // ExpectedListNames is the result expected from a call to `List` when just
 // object names are requested.
-var ExpectedListNames = []string{"hello", "goodbye"}
+var ExpectedListNames = []string{"goodbye", "hello"}
```

Comment:
I am not sure how this ever worked. Objects are always returned lexicographically sorted.